rest/tests/test_learning_resource.py

```diff
@@ -451,3 +547,123 @@ def test_static_assets_delete(self):
             self.DEFAULT_LR_DICT
         )
         self.assertEqual(resp.status_code, HTTP_405_METHOD_NOT_ALLOWED)
+
+    def test_learning_resource_exports(self):
+        """Test for creating learning resource ids in session shopping cart"""
+
+        # setup our data
```

Comment:
"setup" is a noun, "set up" is a verb. This typo is duplicated elsewhere in this diff. Please update.